src/dune_rules/melange/melange_stanzas.ml

```diff
@@ -146,7 +146,7 @@ end
 let syntax =
   Dune_lang.Syntax.create ~name:Dune_project.Melange_syntax.name
     ~desc:"support for Melange compiler"
-    [ ((0, 1), `Since (3, 7)) ]
+    [ ((0, 1), `Since (3, 8)) ]
```

Comment:
This will cause a slightly confusing message for users of melange upgrading to 3.8.0, but that's probably OK given the status of the support. (just a heads-up)